src/Linux/Core/Impl/Microsoft.R.Common.Core.Linux.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard1.6</TargetFramework>
+    <AssemblyName>Microsoft.R.Common.Core.Linux</AssemblyName>
```

Comment:
You can set default namespace here:
`<RootNamespace>Microsoft.Common.Core</RootNamespace>`